<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="6" ignored="24" total="36" passed="6">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="124158" started-at="2021-02-11T17:54:14Z" finished-at="2021-02-11T17:56:18Z">
    <groups>
    </groups>
    <test name="Test" duration-ms="124158" started-at="2021-02-11T17:54:14Z" finished-at="2021-02-11T17:56:18Z">
      <class name="com.tests.UpdateProfile_Test">
        <test-method status="FAIL" signature="UpdateProfile()[pri:0, instance:com.tests.UpdateProfile_Test@2ab4bc72]" name="UpdateProfile" duration-ms="27020" started-at="2021-02-11T17:54:35Z" finished-at="2021-02-11T17:55:02Z">
          <exception class="org.openqa.selenium.NoSuchElementException">
            <message>
              <![CDATA[no such element: Unable to locate element: {"method":"xpath","selector":"//li[contains(text(),'My Profile')]"}
  (Session info: headless chrome=88.0.4324.150)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'VIBHAKUMARI', ip: '192.168.1.12', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_221'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 88.0.4324.150, chrome: {chromedriverVersion: 88.0.4324.96 (68dba2d8a0b14..., userDataDir: C:\Users\VIBHAK~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:55790}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: b92d846e7365b309cffcf2ff8dfdb556
*** Element info: {Using=xpath, value=//li[contains(text(),'My Profile')]}]]>
            </message>
            <full-stacktrace>
              <![CDATA[org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//li[contains(text(),'My Profile')]"}
  (Session info: headless chrome=88.0.4324.150)
For documentation on this error, please visit: https://www.seleniumhq.org/exceptions/no_such_element.html
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'VIBHAKUMARI', ip: '192.168.1.12', os.name: 'Windows 10', os.arch: 'amd64', os.version: '10.0', java.version: '1.8.0_221'
Driver info: org.openqa.selenium.chrome.ChromeDriver
Capabilities {acceptInsecureCerts: false, browserName: chrome, browserVersion: 88.0.4324.150, chrome: {chromedriverVersion: 88.0.4324.96 (68dba2d8a0b14..., userDataDir: C:\Users\VIBHAK~1\AppData\L...}, goog:chromeOptions: {debuggerAddress: localhost:55790}, javascriptEnabled: true, networkConnectionEnabled: false, pageLoadStrategy: normal, platform: WINDOWS, platformName: WINDOWS, proxy: Proxy(), setWindowRect: true, strictFileInteractability: false, timeouts: {implicit: 0, pageLoad: 300000, script: 30000}, unhandledPromptBehavior: dismiss and notify, webauthn:extension:largeBlob: true, webauthn:virtualAuthenticators: true}
Session ID: b92d846e7365b309cffcf2ff8dfdb556
*** Element info: {Using=xpath, value=//li[contains(text(),'My Profile')]}
at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.createException(W3CHttpResponseCodec.java:187)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:122)
at org.openqa.selenium.remote.http.W3CHttpResponseCodec.decode(W3CHttpResponseCodec.java:49)
at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:158)
at org.openqa.selenium.remote.service.DriverCommandExecutor.execute(DriverCommandExecutor.java:83)
at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:323)
at org.openqa.selenium.remote.RemoteWebDriver.findElementByXPath(RemoteWebDriver.java:428)
at org.openqa.selenium.By$ByXPath.findElement(By.java:353)
at org.openqa.selenium.remote.RemoteWebDriver.findElement(RemoteWebDriver.java:315)
at org.openqa.selenium.support.pagefactory.DefaultElementLocator.findElement(DefaultElementLocator.java:69)
at org.openqa.selenium.support.pagefactory.internal.LocatingElementHandler.invoke(LocatingElementHandler.java:38)
at com.sun.proxy.$Proxy9.click(Unknown Source)
at com.Pages.SignIn_Page.clickListProfile(SignIn_Page.java:126)
at com.tests.UpdateProfile_Test.UpdateProfile(UpdateProfile_Test.java:98)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- org.openqa.selenium.NoSuchElementException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- UpdateProfile -->
        <test-method status="PASS" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:com.tests.UpdateProfile_Test@2ab4bc72]" name="closeBrowser" is-config="true" duration-ms="600" started-at="2021-02-11T17:55:02Z" finished-at="2021-02-11T17:55:02Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=UpdateProfile status=FAILURE method=UpdateProfile_Test.UpdateProfile()[pri:0, instance:com.tests.UpdateProfile_Test@2ab4bc72] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
      </class> <!-- com.tests.UpdateProfile_Test -->
      <class name="com.tests.SocialMedia_Test">
        <test-method status="PASS" signature="SocialMedia()[pri:0, instance:com.tests.SocialMedia_Test@3de8f619]" name="SocialMedia" duration-ms="7499" started-at="2021-02-11T17:54:27Z" finished-at="2021-02-11T17:54:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SocialMedia -->
        <test-method status="PASS" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:com.tests.SocialMedia_Test@3de8f619]" name="closeBrowser" is-config="true" duration-ms="1" started-at="2021-02-11T17:54:35Z" finished-at="2021-02-11T17:54:35Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=SocialMedia status=SUCCESS method=SocialMedia_Test.SocialMedia()[pri:0, instance:com.tests.SocialMedia_Test@3de8f619] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="QuitDriver()[pri:0, instance:com.tests.SocialMedia_Test@3de8f619]" name="QuitDriver" is-config="true" duration-ms="78" started-at="2021-02-11T17:54:35Z" finished-at="2021-02-11T17:54:35Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- QuitDriver -->
      </class> <!-- com.tests.SocialMedia_Test -->
      <class name="com.tests.BusHireOperator_Test">
        <test-method status="PASS" signature="BusHireOperator()[pri:0, instance:com.tests.BusHireOperator_Test@2a8448fa]" name="BusHireOperator" duration-ms="6401" started-at="2021-02-11T17:55:53Z" finished-at="2021-02-11T17:55:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- BusHireOperator -->
        <test-method status="PASS" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:com.tests.BusHireOperator_Test@2a8448fa]" name="closeBrowser" is-config="true" duration-ms="0" started-at="2021-02-11T17:55:59Z" finished-at="2021-02-11T17:55:59Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=BusHireOperator status=SUCCESS method=BusHireOperator_Test.BusHireOperator()[pri:0, instance:com.tests.BusHireOperator_Test@2a8448fa] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="QuitDriver()[pri:0, instance:com.tests.BusHireOperator_Test@2a8448fa]" name="QuitDriver" is-config="true" duration-ms="14" started-at="2021-02-11T17:55:59Z" finished-at="2021-02-11T17:55:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- QuitDriver -->
      </class> <!-- com.tests.BusHireOperator_Test -->
      <class name="com.tests.Sites_Test">
        <test-method status="PASS" signature="get_Title()[pri:0, instance:com.tests.Sites_Test@4e5ed836]" name="get_Title" duration-ms="8075" started-at="2021-02-11T17:55:02Z" finished-at="2021-02-11T17:55:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- get_Title -->
        <test-method status="PASS" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:com.tests.Sites_Test@4e5ed836]" name="closeBrowser" is-config="true" duration-ms="1" started-at="2021-02-11T17:55:11Z" finished-at="2021-02-11T17:55:11Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=get_Title status=SUCCESS method=Sites_Test.get_Title()[pri:0, instance:com.tests.Sites_Test@4e5ed836] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="QuitDriver()[pri:0, instance:com.tests.Sites_Test@4e5ed836]" name="QuitDriver" is-config="true" duration-ms="127" started-at="2021-02-11T17:55:11Z" finished-at="2021-02-11T17:55:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- QuitDriver -->
      </class> <!-- com.tests.Sites_Test -->
      <class name="com.tests.Career_Test">
        <test-method status="PASS" signature="setExtent()[pri:0, instance:com.tests.Career_Test@5b799640]" name="setExtent" is-config="true" duration-ms="122" started-at="2021-02-11T17:54:14Z" finished-at="2021-02-11T17:54:14Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setExtent -->
        <test-method status="PASS" signature="endReport()[pri:0, instance:com.tests.Career_Test@5b799640]" name="endReport" is-config="true" duration-ms="215" started-at="2021-02-11T17:56:18Z" finished-at="2021-02-11T17:56:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- com.tests.Career_Test -->
      <class name="com.tests.ManageBooking_Test">
        <test-method status="FAIL" signature="Cancel()[pri:0, instance:com.tests.ManageBooking_Test@eadd4fb]" name="Cancel" duration-ms="6868" started-at="2021-02-11T17:55:11Z" finished-at="2021-02-11T17:55:18Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [Cancel Tichgjket - redBus] but found [Cancel Ticket - redBus]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [Cancel Tichgjket - redBus] but found [Cancel Ticket - redBus]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at org.testng.Assert.assertEquals(Assert.java:463)
at com.tests.ManageBooking_Test.Cancel(ManageBooking_Test.java:52)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Cancel -->
        <test-method status="PASS" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:com.tests.ManageBooking_Test@eadd4fb]" name="closeBrowser" is-config="true" duration-ms="308" started-at="2021-02-11T17:55:18Z" finished-at="2021-02-11T17:55:18Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Cancel status=FAILURE method=ManageBooking_Test.Cancel()[pri:0, instance:com.tests.ManageBooking_Test@eadd4fb] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="FAIL" signature="Reschedule()[pri:0, instance:com.tests.ManageBooking_Test@eadd4fb]" name="Reschedule" duration-ms="4295" started-at="2021-02-11T17:55:18Z" finished-at="2021-02-11T17:55:22Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [true] but found [false]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertTrue(Assert.java:44)
at org.testng.Assert.assertTrue(Assert.java:54)
at com.tests.ManageBooking_Test.Reschedule(ManageBooking_Test.java:84)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Reschedule -->
        <test-method status="PASS" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:com.tests.ManageBooking_Test@eadd4fb]" name="closeBrowser" is-config="true" duration-ms="302" started-at="2021-02-11T17:55:22Z" finished-at="2021-02-11T17:55:22Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Reschedule status=FAILURE method=ManageBooking_Test.Reschedule()[pri:0, instance:com.tests.ManageBooking_Test@eadd4fb] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="FAIL" signature="Show_MyTicket()[pri:0, instance:com.tests.ManageBooking_Test@eadd4fb]" name="Show_MyTicket" duration-ms="6679" started-at="2021-02-11T17:55:22Z" finished-at="2021-02-11T17:55:29Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.tests.ManageBooking_Test.Show_MyTicket(ManageBooking_Test.java:112)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Show_MyTicket -->
        <test-method status="PASS" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:com.tests.ManageBooking_Test@eadd4fb]" name="closeBrowser" is-config="true" duration-ms="302" started-at="2021-02-11T17:55:29Z" finished-at="2021-02-11T17:55:29Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Show_MyTicket status=FAILURE method=ManageBooking_Test.Show_MyTicket()[pri:0, instance:com.tests.ManageBooking_Test@eadd4fb] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="FAIL" signature="SmS()[pri:0, instance:com.tests.ManageBooking_Test@eadd4fb]" name="SmS" duration-ms="6338" started-at="2021-02-11T17:55:29Z" finished-at="2021-02-11T17:55:36Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.tests.ManageBooking_Test.SmS(ManageBooking_Test.java:140)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- SmS -->
        <test-method status="PASS" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:com.tests.ManageBooking_Test@eadd4fb]" name="closeBrowser" is-config="true" duration-ms="299" started-at="2021-02-11T17:55:36Z" finished-at="2021-02-11T17:55:36Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=SmS status=FAILURE method=ManageBooking_Test.SmS()[pri:0, instance:com.tests.ManageBooking_Test@eadd4fb] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="QuitDriver()[pri:0, instance:com.tests.ManageBooking_Test@eadd4fb]" name="QuitDriver" is-config="true" duration-ms="72" started-at="2021-02-11T17:55:36Z" finished-at="2021-02-11T17:55:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- QuitDriver -->
      </class> <!-- com.tests.ManageBooking_Test -->
      <class name="com.tests.Invalid_SignIn">
        <test-method status="FAIL" signature="invalid_SignIn()[pri:0, instance:com.tests.Invalid_SignIn@6f204a1a]" name="invalid_SignIn" duration-ms="18256" started-at="2021-02-11T17:55:59Z" finished-at="2021-02-11T17:56:17Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [false] but found [true]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:568)
at org.testng.Assert.assertEquals(Assert.java:578)
at com.tests.Invalid_SignIn.invalid_SignIn(Invalid_SignIn.java:58)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.apache.maven.surefire.testng.TestNGExecutor.run(TestNGExecutor.java:284)
at org.apache.maven.surefire.testng.TestNGXmlTestSuite.execute(TestNGXmlTestSuite.java:75)
at org.apache.maven.surefire.testng.TestNGProvider.invoke(TestNGProvider.java:119)
at org.apache.maven.surefire.booter.ForkedBooter.runSuitesInProcess(ForkedBooter.java:428)
at org.apache.maven.surefire.booter.ForkedBooter.execute(ForkedBooter.java:162)
at org.apache.maven.surefire.booter.ForkedBooter.run(ForkedBooter.java:562)
at org.apache.maven.surefire.booter.ForkedBooter.main(ForkedBooter.java:548)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- invalid_SignIn -->
        <test-method status="PASS" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:com.tests.Invalid_SignIn@6f204a1a]" name="closeBrowser" is-config="true" duration-ms="665" started-at="2021-02-11T17:56:17Z" finished-at="2021-02-11T17:56:18Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=invalid_SignIn status=FAILURE method=Invalid_SignIn.invalid_SignIn()[pri:0, instance:com.tests.Invalid_SignIn@6f204a1a] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="QuitDriver()[pri:0, instance:com.tests.Invalid_SignIn@6f204a1a]" name="QuitDriver" is-config="true" duration-ms="76" started-at="2021-02-11T17:56:18Z" finished-at="2021-02-11T17:56:18Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- QuitDriver -->
      </class> <!-- com.tests.Invalid_SignIn -->
      <class name="com.tests.SignIn_Test">
        <test-method status="PASS" signature="Valid_SignIn()[pri:0, instance:com.tests.SignIn_Test@239a307b]" name="Valid_SignIn" duration-ms="16526" started-at="2021-02-11T17:55:36Z" finished-at="2021-02-11T17:55:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- Valid_SignIn -->
        <test-method status="PASS" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:com.tests.SignIn_Test@239a307b]" name="closeBrowser" is-config="true" duration-ms="0" started-at="2021-02-11T17:55:53Z" finished-at="2021-02-11T17:55:53Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=Valid_SignIn status=SUCCESS method=SignIn_Test.Valid_SignIn()[pri:0, instance:com.tests.SignIn_Test@239a307b] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="QuitDriver()[pri:0, instance:com.tests.SignIn_Test@239a307b]" name="QuitDriver" is-config="true" duration-ms="31" started-at="2021-02-11T17:55:53Z" finished-at="2021-02-11T17:55:53Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- QuitDriver -->
      </class> <!-- com.tests.SignIn_Test -->
      <class name="com.tests.TermsAndCondition_Test">
        <test-method status="PASS" signature="TAndC()[pri:0, instance:com.tests.TermsAndCondition_Test@6cb107fd]" name="TAndC" duration-ms="5779" started-at="2021-02-11T17:54:14Z" finished-at="2021-02-11T17:54:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TAndC -->
        <test-method status="PASS" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:com.tests.TermsAndCondition_Test@6cb107fd]" name="closeBrowser" is-config="true" duration-ms="4" started-at="2021-02-11T17:54:20Z" finished-at="2021-02-11T17:54:20Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=TAndC status=SUCCESS method=TermsAndCondition_Test.TAndC()[pri:0, instance:com.tests.TermsAndCondition_Test@6cb107fd] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="QuitDriver()[pri:0, instance:com.tests.TermsAndCondition_Test@6cb107fd]" name="QuitDriver" is-config="true" duration-ms="11" started-at="2021-02-11T17:54:20Z" finished-at="2021-02-11T17:54:20Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- QuitDriver -->
      </class> <!-- com.tests.TermsAndCondition_Test -->
      <class name="com.tests.TrackBus_Test">
        <test-method status="PASS" signature="TrackBus()[pri:0, instance:com.tests.TrackBus_Test@710636b0]" name="TrackBus" duration-ms="7140" started-at="2021-02-11T17:54:20Z" finished-at="2021-02-11T17:54:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- TrackBus -->
        <test-method status="PASS" signature="closeBrowser(org.testng.ITestResult)[pri:0, instance:com.tests.TrackBus_Test@710636b0]" name="closeBrowser" is-config="true" duration-ms="1" started-at="2021-02-11T17:54:27Z" finished-at="2021-02-11T17:54:27Z">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=TrackBus status=SUCCESS method=TrackBus_Test.TrackBus()[pri:0, instance:com.tests.TrackBus_Test@710636b0] output={null}]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- closeBrowser -->
        <test-method status="PASS" signature="QuitDriver()[pri:0, instance:com.tests.TrackBus_Test@710636b0]" name="QuitDriver" is-config="true" duration-ms="33" started-at="2021-02-11T17:54:27Z" finished-at="2021-02-11T17:54:27Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- QuitDriver -->
      </class> <!-- com.tests.TrackBus_Test -->
    </test> <!-- Test -->
  </suite> <!-- Suite -->
</testng-results>
